modules/friendlytag.js

```diff
@@ -903,6 +903,10 @@ Twinkle.tag.article.tagList = {
 		'Categories': [
```

Comment:
Cut and paste both 'Categories' and 'Accessibility' so that they are in alphabetical order

Reply:
@NovemLinguae, they have not been active for a year, so can I create a new PR?

Reply:
Sure go ahead. If you can add a comment in your new PR reminding me to close this one, that'd be appreciated.